deepsea.spec.in

```diff
@@ -22,7 +22,7 @@
 # See also http://en.opensuse.org/openSUSE:Shared_library_packaging_policy
 
 Name:           deepsea
-Version:        0.8
+Version:        @VERSION@
 Release:        0
```

Comment:
You might want to check if the (rather complicated) version string you intend to use here is compatible with both DEB and RPM. As I mentioned before (i.e. sorry for repeating myself but it might be important), upstream Ceph chose to split the components by putting only the version number in "Version:" and the rest in "Release:".

I do think this approach might end up being more "universal" but I don't (yet) have any evidence or hard arguments to prove that.

Reply:
@smithfarm thanks for the reminder. I know that in opensuse that version string is currently working good as it's the format already used for DeepSea for several months now.
I'll need to check if that works in other distros, including debian based one.

Reply:
After reading https://www.debian.org/doc/debian-policy/#s-f-version I *think* the full version.txt version number will work as a Debian "upstream_version" number, i.e. without use of "debian_revision". So my comment above can be safely disregarded.

Reply:
@rjfd https://www.debian.org/doc/debian-policy/#s-f-version has a very nice explanation of how Debian does the version comparison. I think you'll be fine.